### PR TITLE
brand and model added to equipments in lendings

### PR DESCRIPTION
**Pull request by MoisesBarbachano**

### CODE REVIEW

File: app/javascript/components/LendingsReturns.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const LendingsReturns = (props) => {
   }
 
   const showList = () => {
-    window.location.reload();
+    //window.location.reload();
```
Comment (Pako7) [inline app/javascript/components/LendingsReturns.js]: Eliminar

File: app/serializers/lending_item_serializer.rb [hunk 2]
```diff
@@ -1,8 +1,16 @@
 class LendingItemSerializer < ActiveModel::Serializer
-  attributes :id, :observation, :lending_id, :status
+  attributes :id, :observation, :lending_id, :status, :equipment_model, :brand
 
   belongs_to :equipment
-  
+
+  def equipment_model
```
Comment (Pako7) [inline app/serializers/lending_item_serializer.rb]: Esto no es correcto.

File: app/javascript/components/LendingDetails.js [hunk 3]
```diff
@@ -109,17 +110,22 @@ const LendingDetails = (props) => {
...
 
   const content = equipments.map((equipment) => {
     const { id, description, serie} = equipment.equipment;
+    const model = equipment.equipment_model.name;
+    const brand = equipment.brand.name;
+
     return(
       <tr key={id}>
-        <Td>{description}</Td>
+        <Td>{brand}</Td>
```
Comment (ChemaPool) [inline app/javascript/components/LendingDetails.js]: Cambiar orden

File: app/javascript/components/LendingDetails.js [hunk 4]
```diff
@@ -109,21 +114,71 @@ const LendingDetails = (props) => {
...
+        <Td>
+          <Link id={`btnModal${index}`} onClick={event => openModal(event, index)}>
+            Ver más
+          </Link>
+        </Td>
       </tr>
     )
   });
 
+  //setObservations(observationsEquipments);
```
Comment (Arcanm) [inline app/javascript/components/LendingDetails.js]: quitar

**Pull request closed without merging**